system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
+Documentation=man:netdata
+Documentation=file:///usr/share/doc/netdata/html/index.html
+Documentation=https://github.com/firehol/netdata
 After=network.target httpd.service squid.service nfs-server.service mysqld.service named.service postfix.service
+Wants=network-online.target
+ConditionPathExists=/etc/netdata/netdata.conf
 
 [Service]
 Type=simple
+Environment="netdata_LOG_LOCATION=/var/log/netdata/log"
+ExecStart=@sbindir_POST@/netdata -D
+ExecReload=@sbindir_POST@/netdata reload
+TimeoutStopSec=60
+KillMode=mixed
+KillSignal=SIGTERM
+OOMScoreAdjust=-900
```

Comment:
netdata adjusts its OOMScore to 1000 already: https://github.com/firehol/netdata/blob/4f0d3af03cbc6431a0083448dc4dc127b5459f51/src/daemon.c#L242

According to systemd docs:

> Sets the adjustment level for the Out-Of-Memory killer for executed processes. Takes an integer between -1000 (to disable OOM killing for this process) and 1000 (to make killing of this process under memory pressure very likely).

So, why to make netdata unlikely to be killed by OOM? It should be likely to be killed by OOM. We don't want to have our monitoring running but the application servers crashed...